booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -508,6 +508,33 @@ private static JsonNode getShipmentLocationTypeCode(JsonNode body, @NonNull Stri
       return issues;
     });
 
+  static final JsonContentCheck CHECK_CARGO_GROSS_WEIGHT_CONDITIONS =
```

Comment:
As part of this issue, shouldn't we validate the other way too that if it's not present at the commodities level it should be mandatory at RE level or should the message be changed like that?